UnitsNet.Tests/CustomCode/HeatFluxTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated (once) by \generate-code.bat, but will not be
+//     regenerated when it already exists. The purpose of creating this file is to make 
+//     it easier to remember to implement all the unit conversion test cases.
+// 
+//     Whenever a new unit is added to this quantity and \generate-code.bat is run,
+//     the base test class will get a new abstract property and cause a compile error
+//     in this derived class, reminding the developer to implement the test case
+//     for the new unit.
+//
+//     See https://github.com/angularsen/UnitsNet/wiki/Adding-a-New-Unit for how to add or edit units.
+//
+//     Add CustomCode\Quantities\MyQuantity.extra.cs files to add code to generated quantities.
+//     Add Extensions\MyQuantityExtensions.cs to decorate quantities with new behavior.
+//     Add UnitDefinitions\MyQuantity.json and run GeneratUnits.bat to generate new units or quantities.
+//
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+// Copyright (c) 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com).
+// https://github.com/angularsen/UnitsNet
+// 
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+// 
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+
+using System;
+using Xunit;
+
+namespace UnitsNet.Tests.CustomCode
+{
+    public class HeatFluxTests : HeatFluxTestsBase
+    {
+        protected override double BtuPerSquareFootHourInOneWattPerSquareMeter => 3.16998331e-1;
```

Comment:
That is correct, this is the international table BTU, not thermochemical. Do you distinguish anywhere? I did not see any examples. I would generally prefer to stick with it. To quote the [American Physical Society](https://www.aps.org/policy/reports/popa-reports/energy/units.cfm):

>The International Table (IT) calorie has been adopted in the publications of the Energy Information Administration of the U.S. Department of Energy (DOE/EIA) (3) and of the International Energy Agency of the Organization for Economic Co-operation and Development (OECD/IEA) (4). In view of the importance of these publications, it is reasonable to view the IT calorie as being the preferred unit for discussions of energy production and use, but there is no universally adopted practice (see also the discussion of Btu, below).


 I think you are correct about the ordering with names. I will change this.

Reply:
I'm no domain expert and not really familiar with these IT and Th variants myself.
We don't have a guideline on these yet, but here is what we have so far:

* SpecificEnergy.CaloriePerGram
* SpecificEntropy.CaloriePerGramKelvin
* Energy.Calorie

So we don't specify variant for calories already. Not sure if these are IT or Th, or what is the most widely used.

I don't find any mention of IT and Th in existing units, so I may have been confused by the `Therm` unit with abbreviation `th`. 

I recall these after some discussions and googling in earlier issues, but searching I only found these so I guess we haven't thought too much about it yet:
https://github.com/angularsen/UnitsNet/pull/322#discussion_r151859232 (SpecificEntropy)
https://github.com/angularsen/UnitsNet/issues/85 (BTU)

So. If I am to conclude on anything => no, we don't currently specify IT or Th and we don't distinguish them. The question is, should we? Would you know what makes the most sense?

Reply:
Since it seems we have been using IT before and you also find that the most natural, I suggest we stick with it until someone comes around and actually needs the Th variant and a way to distinguish them.